go/billing/models.py

```diff
@@ -216,10 +216,22 @@ class Transaction(models.Model):
         (STATUS_REVERSED, STATUS_REVERSED),
     )
 
+    TRANSACTION_TYPE_MESSAGE = 'Message'
+    TRANSACTION_TYPE_TOPUP = 'Top Up'
+    TRANSACTION_TYPE_CHOICES = (
+        (TRANSACTION_TYPE_TOPUP, TRANSACTION_TYPE_TOPUP),
+        (TRANSACTION_TYPE_MESSAGE, TRANSACTION_TYPE_MESSAGE),
```

Comment:
I think I have a vague preference for the database values to be `"message"` and `"topup"` rather than the same as the labels (just feels more like what I'd expect and means less remembering capitalization and spacing).